mars_street_view/scripts/initializedb.py

```diff
@@ -7,34 +8,108 @@
 from pyramid.paster import (
     get_appsettings,
     setup_logging,
-    )
+)
 
 from pyramid.scripts.common import parse_vars
 
 from ..models import (
     DBSession,
-    MyModel,
     Base,
-    )
+    Rover,
+    Camera
+)
 
 
 def usage(argv):
+    """Print message to stdout explaining correct usage syntax."""
     cmd = os.path.basename(argv[0])
     print('usage: %s <config_uri> [var=value]\n'
           '(example: "%s development.ini")' % (cmd, cmd))
     sys.exit(1)
 
 
 def main(argv=sys.argv):
+    """Initialize database, optionally with settings from config uri."""
     if len(argv) < 2:
         usage(argv)
     config_uri = argv[1]
     options = parse_vars(argv[2:])
     setup_logging(config_uri)
     settings = get_appsettings(config_uri, options=options)
+    # import pdb; pdb.set_trace()
+    if not settings.get('sqlalchemy.url'):
+        try:
+            settings['sqlalchemy.url'] = os.environ['MARS_DATABASE_URL']
+        except KeyError:
+            print('Required MARS_DATABASE_URL not set in global os environ.')
+            sys.exit()
+        try:
+            settings['nasa_api_key'] = os.environ['NASA_API_KEY']
+        except KeyError:
+            print('Required NASA_API_KEY not set in global os environment.')
+            sys.exit()
```

Comment:
given that this appears twice, it might be worth factoring it into a stand-alone function that throws an error.  Then you could update it in only one place the next time you have another env variable you depend on.